backend.native/tests/build.gradle

```diff
@@ -317,8 +317,8 @@ def createTestTasks(File testRoot, Class<Task> taskType, Closure taskConfigurati
 
 void dependsOnPlatformLibs(Task t) {
     if (!useCustomDist) {
-        def testTarget = project.testTarget
-        if (testTarget != null && testTarget != project.hostName) {
+        def testTarget = project.target
```

Comment:
Why this change is needed?

Reply:
It was for running raspberry-pi tests. @PavelPunegov should commit sane changes on master.